index.html

```diff
@@ -1,10 +1,248 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link href="css/styles.css" rel="stylesheet" />
-    <title>Document</title>
-  </head>
-  <body></body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="css/styles.css" rel="stylesheet" />
+  <title>Swiss Bagel Salad</title>
+</head>
+
+<body>
+  <div class="wrapper">
+    <div class="upper-box">
+      <!-- title -->
+      <div class="title-wrapper background">
+        <div class="swiss">
+          <h1>Swiss</h1>
+        </div>
+        <div class="bagel">
+          <h1>bagel</h1>
+        </div>
+        <div class="salad">
+          <h1>salad</h1>
+        </div>
+      </div>
+      <div class="title-wrapper text">
+        <div class="swiss">
+          <h1>Swiss</h1>
+        </div>
+        <div class="bagel">
+          <h1>bagel</h1>
+        </div>
+        <div class="salad">
+          <h1>salad</h1>
+        </div>
```

Comment:
Hello @angeliquejw!

Thank you for your comments! 
I didn't know the fact that div could be wrapped by h1. Good to know that!
I don't think I understand the last sentence though. Could you explain more about it?

Reply:
Yes, a heading element is like a `div` or a `p` element in that it can still have other HTML elements inside of it 🙌 

Screen readers read all the content on the page -- so, even though you've visually overlapped the text, you still have the following text on your page:
Swiss bagel salad
Swiss bagel salad

A someone looking at the site doesn't know the words are there twice, but someone using any type of screen reader software would hear the words twice -- which is confusing and not ideal.

The attribute `aria-hidden` tells a screen reader to ignore the element. So, I'd recommend your HTML be updated to:

```html
<div class="title-wrapper background">
  <h1>
    <div class="swiss">Swiss</div>
    <div class="bagel">bagel</div>
    <div class="salad">salad</div>
  </h1>
</div>
<div class="title-wrapper text">
  <h1 aria-hidden="true">
    <div class="swiss">Swiss</div>
    <div class="bagel">bagel</div>
    <div class="salad">salad</div>
  </h1>
</div>
```

You can read more about `aria-hidden` in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute)